test/sqllogictest/explain.slt

```diff
@@ -118,29 +112,16 @@ EOF
 query T multiline
 EXPLAIN TYPED DECORRELATED PLAN FOR VIEW foo
 ----
-%0 =
+%0 = Let l0 =
 | Constant ()
-| | types = ()
-| | keys = ()
 
 %1 =
-| Get %0
-| | types = ()
-| | keys = ()
-
-%2 =
 | Constant ()
-| | types = ()
-| | keys = ()
 
-%3 =
-| Join %1 %2
+%2 =
+| Join %0 %1
 | | implementation = Unimplemented
-| | types = ()
-| | keys = ()
 | Map 1
-| | types = (i32)
-| | keys = ()
```

Comment:
Did something bad happen with these types and keys going away? I'm not super familiar so maybe intended?

Reply:
Oops!

Reply:
Ok, fixed! Good catch!